harness/engine/utils.go

```diff
@@ -70,10 +70,10 @@ func (n *Network) SendQuery(nodeIndex uint, query string, result interface{}) er
 }
 
 // SendCommand sends a jsonrpc request to the specified network node
```

Comment:
add info about the semantics of the returned string. What should the client of `SendCommand` expect?